Source/WebKit/UIProcess/WebsiteData/Cocoa/WebsiteDataStoreCocoa.mm

```diff
@@ -336,6 +323,37 @@ static String defaultWebsiteDataStoreRootDirectory()
     });
 }
 
+void WebsiteDataStore::removeDataStoreWithIdentifier(const WTF::UUID& identifier, CompletionHandler<void(const String&)>&& callback)
+{
+    ASSERT(isMainRunLoop());
+
+    auto completionHandler = [identifier, callback = WTFMove(callback)](const String& error) mutable {
+        RELEASE_LOG(Storage, "WebsiteDataStore::removeDataStoreWithIdentifier: Removal completed for identifier %s (error '%s')", identifier.toString().utf8().data(), error.isEmpty() ? "null"_s : error.utf8().data());
```

Comment:
Strings need to be logged with PUBLIC_LOG_STRING if there are not sensitive and if you want them to show in sysdiagnoses.